core/templates/dev/head/pages/exploration_editor/translation_tab/audio_translation_bar_directive.html

```diff
@@ -180,6 +188,11 @@
     <strong>Warning: </strong>Don't navigate to other tabs of this page before saving recorded audio, otherwise the recorded audio will be lost.
   </span>
 </div>
+<div class="oppia-translation-bottom-right-container" ng-if="!showRecorderWarning && !audioBlob ">
+  <span>
+    <strong>ProTip: </strong>Use key "R" to start/stop recording
```

Comment:
Change the color of this message to black (It's red right now which I think indicates a warning.)

Reply:
Has this been addressed? Please make sure to reply to all comments. Thanks!

Reply:
Done